### ISSUE TITLE
Build fails: don't know how to make install/strip. Stop

### ISSUE THREAD

**Issue by yurivict** [horde3d/Horde3D#109]
Both 1.0 and current revisions fail:
```
make[1]: don't know how to make install/strip. Stop
```

I'm pretty sure something's wrong with your cmake makefiles.

On FreeBSD.

**Comment by horde3d** [comment]
Well depends on what you define with "wrong".
I guess CMAKE_SYSTEM_NAME is FreeBSD on your system, while we only support Linux.
So maybe you can fix it by extending all comparison of CMAKE_SYSTEM_NAME with Linux with an OR  ${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD" 
Unfortunately I don't have a FreeBSD system available so I not test it myself

**Comment by yurivict** [comment]
You are right. For compatibility with BSDs, you need to change them all to ```${CMAKE_SYSTEM_NAME} MATCHES "Linux|.*BSD"```.

Besides this, it failed because ```-fPIC``` was missing. It succeeded after I added ```-fPIC```.
